pagopa/io-app
# chore: [IOPID-2538] Fix `LOGIN_CIEID_SELECTED` event delay

## Short description
This PR removes the delay in sending some login method selection events.
>[!Note]
>This fix resolves the current behavior where the `LOGIN_CIEID_SELECTED` event is sent after `LOGIN_CIEID_APP_NOT_FOUND`, even though the latter occurs later in the flow

## List of changes proposed in this pull request
- Moved the `updateMixpanelProfileProperties` async action after `mixpanelTrack` to ensure the event order accurately reflects the intended flow

## Demo
### Before
https://github.com/user-attachments/assets/a2cf58de-0fdb-45a7-a7cc-347a5f9929fc

### After
https://github.com/user-attachments/assets/a6449720-d40d-451c-9936-4dd754f53091

## How to test
Run the app and emulate what shown in the demo videos to ensure that the event dispatch order is respected.
